_publications/turilli2018building

```diff
@@ -0,0 +1,14 @@
+M. Turilli, A. Merzky, V. Balasubramanian, S. Jha. “Building Blocks for Workflow Sys-tem Middleware”. Accepted in Proceedings of the 18th IEEE/ACM International Sym-posium on Cluster, Cloud and Grid Computing (CCGrid), 2018.
```

Comment:
This line blocks the file from being rendered. The header should be the first thing. I am not sure if you want it or not.

Reply:
Apologies, it was a left over from a C&P